documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md

```diff
@@ -221,18 +222,21 @@ The final version of the evaluation report may only be submitted to the Applican
 8. The Applicant may only make the evaluation report available to third parties in full text, stating the date of issue, and pass it on after written approval by the Evaluation Facility.
 
 ### 3.6 Change Certification     
```

Comment:
so far, you only updated §3.6 - please also check **§4.9 Change certification** and **§5.3 Period of validity. Renewed certification** (which should be replaced with "renewal certification" to have a consistent wording)
It might make sense to move some of the newly added information there, since chapter 3 is supposed to focus on the evaluation procedures whereas e.g., the validity period was originally specified in 5.3...

Reply:
Both §4.9 and §5.3 have been updated